pkg/config/config.go

```diff
@@ -59,7 +59,10 @@ func (c *Config) ApplyAliasConfig(parameters []StepParameters, filters StepFilte
 func setParamValueFromAlias(configMap map[string]interface{}, filter []string, p StepParameters) map[string]interface{} {
 	if configMap != nil && configMap[p.Name] == nil && sliceContains(filter, p.Name) {
 		for _, a := range p.Aliases {
-			configMap[p.Name] = getDeepAliasValue(configMap, a.Name)
+			aliasVal := getDeepAliasValue(configMap, a.Name)
+			if aliasVal != nil {
+				configMap[p.Name] = getDeepAliasValue(configMap, a.Name)
```

Comment:
why not setting `aliasVal` into `configMap[p.Name]`?

Reply:
good point, missed that one ...